src/Command/WarbandNotify.js

```diff
@@ -0,0 +1,18 @@
+const Command = require('../Command');
+
+module.exports = Command.extend({
+    commandName: 'warbandnotify',
+    commandAliases: ['notifywarband', 'wbnotify', 'war'],
+    advertisable: false,
+    processMessage: function (message, tokens) {
+        message.delete();
+
+        message.member.roles.add(message.guild.roles.cache.find(role => role.name === 'Warband Playtester'))
+            .catch(error => {
+                message.channel.send(this.i18n.__mf('I was unable to grant you the role for some reason. My permissions may be messed up.'))
```

Comment:
Needs to also be present on the en.json locale file, to ensure that it is not missed when localizing.